### ISSUE TITLE
External Channel Support for Attachments

### ISSUE THREAD

**Issue by tybritten** [nyaruka/courier#340]
Not sure what's the most appropriate way to surface it, but it would be nice to pass mediaurls for images etc to external channels.

**Comment by rowanseymour** [comment]
Makes sense. Probably best if it's implemented the same as quick replies, where `{quick_replies}` becomes either a JSON, XML or querystring representation of all replies depending on the configured content type for the channel

**Comment by rowanseymour** [comment]
Go for it
